File: rest-service/queue/src/main/java/mb/lib/extern/JobCreateResponse.java
package mb.lib.extern;

import com.fasterxml.jackson.annotation.JsonGetter;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonSetter;

@JsonIgnoreProperties(ignoreUnknown = true)
public class JobCreateResponse
{
  private int id;

  @JsonGetter("id")
  public int getId() {
    return id;
  }

  @JsonSetter("id")
  public JobCreateResponse setId(int id) {
    this.id = id;
    return this;
  }
}

File: lib-jvm-mblast-sdk/src/main/kotlin/org/veupathdb/lib/mblast/sdk/report/model/JobCreateResponse.kt
package org.veupathdb.lib.mblast.sdk.report.model

import com.fasterxml.jackson.annotation.JsonProperty
import org.veupathdb.lib.hash_id.HashID

data class JobCreateResponse(
  @JsonProperty("reportJobID")
  val reportJobID: HashID,
)
